nifi-nar-bundles/nifi-azure-bundle/nifi-azure-processors/src/main/java/org/apache/nifi/processors/azure/eventhub/ConsumeAzureEventHub.java

```diff
@@ -332,6 +347,26 @@ public void setWriterFactory(RecordSetWriterFactory writerFactory) {
                     .valid(false)
                     .build());
         }
+
+        if (!StringUtils.isNotBlank(storageAccountKey) && !StringUtils.isNotBlank(storageSasToken)) {
```

Comment:
I think !StringUtils::isNotBlank can be replaced with StringUtils::isBlank.

Reply:
Based on the previous approach, I believe just `StringUtils.isNotBlank()` should be used:
```suggestion
        if (StringUtils.isNotBlank(storageAccountKey) && StringUtils.isNotBlank(storageSasToken)) {
```

Reply:
Agreed.

Reply:
Thanks @Lehel44 code updated accordingly.